costum_events/models/myevents.py

```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
 from odoo import models, fields, api
-class myevents(models.Model):
-     
+class Myevent(models.Model):
      _inherit = 'event.event'
+     _rec_name = 'myevents'
```

Comment:
@mohammed-almaqdy _rec_name takes the name of one of the fields and **myevents** it not fields name